### PR TITLE
Fix for run_quietly issue

### PR DESCRIPTION
**Pull request by collinjc**
Fix for https://github.com/microsoft/mdatp-xplat/issues/25. Also cleaned up whitespace.

### CODE REVIEW

File: linux/installation/mde_installer.sh [hunk 1]
```diff
@@ -108,35 +108,39 @@ run_quietly()
         exit 1
     fi
 
-    local out=$(eval $1 2>&1; echo "$?")
-    local exit_code=$(echo "$out" | tail -n1)
+    if [ "$ASSUMEYES" == "-y" ]; then
```
Comment (MatiAtMicrosoft) [inline linux/installation/mde_installer.sh:111]: So if the condition `"$ASSUMEYES" == "-y"` is not satisfied, nothing happens?

Comment (MatiAtMicrosoft) [inline linux/installation/mde_installer.sh:111]: So if the condition `"$ASSUMEYES" == "-y"` is not satisfied, nothing happens?

Comment (collinjc) [reply linux/installation/mde_installer.sh:111]: No, if the condition is not satisfied, the else condition is used, which does not suppress the output, allowing the user to respond to any prompts which are currently being hidden by the output suppression. Without this change in place (or something like it), anything that is passed to run_queitly that expects user input (package manager installations, for example) are hanging indefinitely waiting for the user to respond yes. As of right now, without this change, the only way the installer successfully runs is if the user passes -y.

**Comment by collinjc** [comment]
Closes #25

**Pull request merged**